frontend/src/mock-data/api.ts

```diff
@@ -1,6 +1,7 @@
 import * as _ from 'lodash';
 import { Subject, Observable, of } from 'rxjs';
 import { mockUserResponse } from './user';
+import { LimitedResultsDownloadParameters } from '../app/models';
```

Comment:
Where exactly is this defined? Can you import from the specific model file for clarity? Given on the name of the type, I cannot quite fathom what this type describes.

Reply:
I agree it would be clearer to remove `app/models/index.ts` (and some similar files) but that's a project configuration. I'll update the import here, in any case.

Reply:
It describes the parameters included in the body in a request for a download with a limited number of results (as opposed to the large download).